src/crc-start.ts

```diff
@@ -88,6 +88,10 @@ export async function startCrc(
 }
 
 async function askAndStorePullSecret(logger: extensionApi.Logger): Promise<boolean> {
+  extensionApi.window.showInformationMessage(`To pull container images from the registry, a pull secret is necessary. You can get a pull secret from the Red Hat OpenShift Local download page by navigating to the dashboard and clicking the "Obtain pull-secret" or opening the following address in your browser https://cloud.redhat.com/openshift/create/local
```

Comment:
`await` before `extensionApi.window.showInformationMessage` helps do not show `extensionApi.window.showInputBox` until user click on `Continue`

Reply:
I considered this... but in this way it is more obvious the dialog is in focus.